packages/ses/types.d.ts

```diff
@@ -36,6 +36,10 @@ export interface LockdownOptions {
   __hardenTaming__?: 'safe' | 'unsafe';
 }
 
+export type LockdownOptions = RepairOptions;
```

Comment:
Please comment that we preserve `LockdownOptions` only for legacy compat. Perhaps deprecate?